classes/class-p4-master-site.php

```diff
@@ -510,7 +510,9 @@ public function enqueue_public_assets() {
 
 		// JS files.
 		wp_register_script( 'jquery', 'https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js', [], '3.3.1', true );
-		wp_enqueue_script( 'main', $this->theme_dir . '/assets/build/index.js', [ 'jquery' ], $js_creation, true );
+		wp_register_script( 'lazyload', 'https://cdn.jsdelivr.net/npm/vanilla-lazyload@11.0.4/dist/lazyload.min.js', [], '1.0.4', true );
```

Comment:
it seems to be available on [cdnjs](https://cdnjs.com/libraries/vanilla-lazyload), so we only fetch stuff from one CDN.

Reply:
Updated.